docs/getting_started/index.md

```diff
@@ -5,46 +5,35 @@ hide:
 
 # Getting Started with Taipy
 
-Welcome to Taipy! This guide will show you how to install Taipy and create your first application.
+Dive into Taipy with this beginner-friendly guide. Learn to install, configure, and deploy your first application with ease.
 
-## Install Taipy with pip
-
-To begin with Taipy, ensure you have Python installed (version 3.8 or higher) along with
-[pip](https://pip.pypa.io). Once you have these prerequisites, open your terminal or command 
-prompt and run the following command to get the latest stable version of Taipy downloaded and 
-installed:
+## Installation
 
+1. **Prerequisites**: Ensure you have Python (version 3.8 or higher) and [pip](https://pip.pypa.io) installed.
+2. **Installation Command**: Run the following in your terminal or command prompt:
 ``` console
 $ pip install taipy
 ```
 
 !!! info "Other installation options"
-    
-    You can find additional instructions on the [installation page](../installation/index.md) 
-    if you require different ways to install Taipy or don't have Python or pip installed.
 
+    For alternative installation methods or if you're lacking Python or pip, refer to the [installation page](../installation/index.md).
 
-## First Taipy Scenario 
-Taipy introduces the concept of a *Scenario* to model pipeline executions. A *Scenario* consists 
-of an execution graph, often represented as a Directed Acyclic Graph (DAG), where tasks (or 
-functions) exchange data. You can make your scenario as easy or as complicated as you want, 
-depending on what you need.
+## Dive into Your First Taipy Scenario
```

Comment:
You already used the metaphor "Dive into" in the very first line. I believe it s repetitive.